k8s_handle/templating.py

```diff
@@ -113,6 +116,40 @@ def generate_by_context(self, context):
                 raise TemplateRenderingError('Unable to render {}, due to: {}'.format(template, e))
         return output
 
+    def _filter_tagged_templates(self, templates, only_tags, skip_tags):
+        return [i for i in templates if self._evaluate_tags(i, only_tags, skip_tags)]
+
+    def _evaluate_tags(self, template, only_tags, skip_tags):
+        if 'tags' in template:
```

Comment:
I think we need to separate template tags analysis and skipping logic to 2 different functions.

Reply:
I moved first part into `_get_template_tags()`.